### PR TITLE
feat(FEC-11889): upgrade to hlsjs 1.1.5

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

Solves FEC-11889

Related to https://github.com/kaltura/playkit-js-hls/pull/167

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "@playkit-js/playkit-js-dash": "1.30.0",
     "@playkit-js/playkit-js-hls": "1.29.0",
     "@playkit-js/playkit-js-ui": "0.70.1",
-    "hls.js": "^1.0.7",
+    "hls.js": "^1.1.5",
```
Comment (SivanA-Kaltura) [inline package.json:64]: why do we actually have this both here and in the adapter ?

Comment (yairans) [reply package.json:64]: it's a good question. I think the idea was to be able to upgrade the 3th party (hlsjs/shaka) without updating the adapter, but in fact we dont do it

Comment (SivanA-Kaltura) [reply package.json:64]: so can we remove it ?

Comment (yairans) [reply package.json:64]: @OrenMe WDYT?

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**